node/external/transactionAPI/apiTransactionProcessor.go

```diff
@@ -103,6 +105,51 @@ func (atp *apiTransactionProcessor) GetTransactionsPool() (*common.TransactionsP
 	return txsPoolResponse, nil
 }
 
+// GetTransactionsPoolForSender will return a structure containing the transactions for sender that is to be returned on API calls
+func (atp *apiTransactionProcessor) GetTransactionsPoolForSender(sender string) (*common.TransactionsPoolForSenderApiResponse, error) {
```

Comment:
in future refactors, we could have something like:
`apiTransactionProcessor.GetTransactionsPool(options GetTxsPoolOptions)` where GetTxsPoolOptions could contain `WithSender string`, `WithReceiver string`, `WithFields []string` and so on.

Reply:
noted, new PRs will be created for this